test/functional/FunctionalLiquidSpec.hs

```diff
@@ -86,13 +86,16 @@ spec = describe "liquid haskell diagnostics" $ do
 
         -- docItem <- getDocItem file languageId
         sendNotification TextDocumentDidSave (DidSaveTextDocumentParams doc)
-        diags2hlint <- waitForDiagnostics
-        -- liftIO $ show diags2hlint `shouldBe` ""
-
-        -- We turned hlint diagnostics off
-        liftIO $ length diags2hlint `shouldBe` 0
-        diags2liquid <- waitForDiagnostics
-        liftIO $ length diags2liquid `shouldBe` 0
+        -- TODO: what does that test?
+        -- TODO: whether hlint is really disbabled?
+        -- TODO: @fendor, document or remove
+        -- diags2hlint <- waitForDiagnostics
+        -- -- liftIO $ show diags2hlint `shouldBe` ""
+
+        -- -- We turned hlint diagnostics off
+        -- liftIO $ length diags2hlint `shouldBe` 0
+        -- diags2liquid <- waitForDiagnostics
+        -- liftIO $ length diags2liquid `shouldBe` 0
```

Comment:
Does someone want to refactor this test-case before merge? I dont understand what is happening, thus all these comment.

Reply:
Perhaps add it to the list of post-merge actions, and allocate it to me.

Reply:
Great, will do